pkg/handler/loki.go

```diff
@@ -181,7 +186,7 @@ func manageStreams(streams loghttp.Streams, columns []string) ([][]string, error
 		if labels == nil {
 			labels = make([]string, 0, len(stream.Labels))
 			for name := range stream.Labels {
-				if columns == nil || contains(columns, name) {
+				if columns == nil || utils.Contains(columns, name) {
```

Comment:
If we have to run this operation multiple times for each entry of each stream, maybe I'd keep all the columns in a `map[string]struct{}` instance so the lookup is O(1) instead of O(n).

Reply:
This is actually run only when `labels == nil` = once unless you don't have labels in some streams but you should not

Reply:
But I can still modify it if you feel it's more readable ?

Reply:
I was just concerned about performance but didn't realized that this was executed only once. Let's keep it as now if it looks more readable.

Reply:
sure, thanks !